packages/code-editor/src/CodeEditorCollection.js

```diff
@@ -23,6 +23,29 @@ export default class CodeEditorCollection extends PureComponent {
     modelSessionManager.editorContainer = this
     this.tabs = React.createRef()
     this.editorContainer = React.createRef()
+    this.tabContextMenu = [
+      {
+        text: 'Close',
+        onClick: this.closeCurrentFile
+      },
+      {
+        text: 'Close Others',
+        onClick: this.closeOtherFiles
+      },
+      {
+        text: 'Close All',
+        onClick: this.closeAll
+      },
+      null,
+      {
+        text: 'Copy Path',
+        onClick: this.copyPath
+      },
+      ...platform.isDesktop ? [{
+        text: 'Reveal In Finder',
```

Comment:
Open Containing Folder
see context menu for the file tree

Reply:
updated: https://github.com/ObsidianLabs/electron-components/pull/56/files#diff-06ff2e4ab637b04f57df25a54798f2a4793cc05670bdf87abe1c3ac9e196bb29R45